src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -798,6 +801,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		coordinator.showOnboarding()
 	}
 
+	private func checkIfBoosterRulesShouldBeFetched() {
+		if let lastExecutionDate = store.lastBoosterNotificationsExecutionDate,
```

Comment:
This check is implemented twice. Can it be moved to the service, so its only implemented at one place?